definitions/ext-rf_scanner/summary_metrics.yml

```diff
@@ -0,0 +1,48 @@
+msToFirstRead:
+  title: Request Time (ms)
+  
+  query:
+    select: average(msToFirstRead)
+    from: THD_RF_TRANSACTION
+    facet: macAddress
+    
+msInRead:
+  title: Response Time (ms)
+  
+  query:
+    select: average(msInRead)
+    from: THD_RF_TRANSACTION
+    facet: macAddress
```

Comment:
Summary metrics don't have a `facet` https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/summary_metrics.md#nrdb-query but they have a mandatory `eventId` field which you can set to `entity.guid`